README.md

```diff
@@ -46,6 +46,7 @@ Simple Router For Queues
 ``` js
 {
    "queue": { // object, required
+      "ackOnMessageError": "" // boolean, optional, (send ack when handle funcrion failed, default true)
```

Comment:
funcrion => function. Also, if its boolean dont set empty string in the esxmaple

Reply:
👍